astropy/io/fits/hdu/hdulist.py

```diff
@@ -43,7 +43,7 @@ def fitsopen(name, mode='readonly', memmap=None, save_backup=False,
         append (ab+), ostream (w), denywrite (rb)).
 
     memmap : bool, optional
-        Is memory mapping to be used?
+        Is memory mapping to be used? ``memmap=True`` by default. This value is obtained from the configuration item ``astropy.io.fits.Conf.use_memmap``.
```

Comment:
Nitpick: Please break long line into 2-3 lines (as needed).

p.s. Perhaps this could have been folded in #7432 but this is fine as standalone too.